ssh/README.md

```diff
@@ -19,6 +19,7 @@ ha help
 
 Follow these steps to get the add-on installed on your system:
 
+0. This addon is only enabled for "Advanced Mode" users. To enable: **Profile** -> **Advanced Mode** 
```

Comment:
Please re-number the other items on the list as well.

```suggestion
`1. This addon is only visible to "Advanced Mode" users. To enable advanced mode, go to **Profile** -> and turn on **Advanced Mode**.
```

Reply:
You've only partly applied my suggestion. Any specific reason?

Reply:
Sorry wasn't paying attention 😖